src/geo/transform.js

```diff
@@ -76,6 +76,13 @@ class Transform {
     pixelMatrix: Float64Array;
     pixelMatrixInverse: Float64Array;
 
+    // From world coordinates to, relative-to-center coordinates.
+    // Relative-to-center coordinates are centered at the current map center, and rotated by the map bearing.
+    // +x basis goes towards the right, -y basis point away from the center, +z basis points up from the map.
```

Comment:
`-y basis point away from the center`
Does this mean that negative values are in front of the camera/center and positive values are behind?

Reply:
Good catch, I forgot to update this comment!